class-activity-2.Rmd

```diff
@@ -76,25 +76,33 @@ pairs(D5)
 1. Create a simulated data set containing 100 students, each with a score from 1-100 representing performance in an educational game. The scores should tend to cluster around 75. Also, each student should be given a classification that reflects one of four interest groups: sport, music, nature, literature.
 
 ```{r}
-#rnorm(100, 75, 15) creates a random sample with a mean of 75 and standard deviation of 20
+#rnorm(100, 75, 15) creates a random sample with a mean of 75 and standard deviation of 15
 #pmax sets a maximum value, pmin sets a minimum value
 #round rounds numbers to whole number values
 #sample draws a random samples from the groups vector according to a uniform distribution
-
-
+studentPerformance = rnorm(100, 75, 15)
+studentPerformance[studentPerformance > 100] = 100
```

Comment:
There is an another way of setting maximum and minimum values with a single line of code. `round(pmax(1, pmin(100, rnorm(n=100, mean=75, sd=15))))`